### PR TITLE
proposal: audit log cleanup

### PR DESCRIPTION
**Pull request by n1ru4l**
A small set of proposals for cleaning up the audit log PR https://github.com/graphql-hive/console/pull/5530.

### CODE REVIEW

File: packages/services/api/src/modules/audit-logs/module.graphql.ts [hunk 1]
```diff
@@ -1,56 +1,22 @@
...
-
-  type AuditLogIdRecord {
-    userId: ID!
-    userEmail: String!
-    organizationId: ID!
-    """
-    User can be null if the user is deleted
-    """
-    user: User
-  }
```
Comment (n1ru4l) [inline packages/services/api/src/modules/audit-logs/module.graphql.ts:23]: These types are unused.

File: packages/services/api/src/modules/audit-logs/module.graphql.ts [hunk 2]
```diff
@@ -1,56 +1,22 @@
...
-      selector: OrganizationSelectorInput!
-      filter: AuditLogFilter!
+      input: ExportOrganizationAuditLogInput!
     ): ExportOrganizationAuditLogResult!
   }
 
+  input ExportOrganizationAuditLogInput {
+    selector: OrganizationSelectorInput!
+    filter: AuditLogFilter!
+  }
```
Comment (n1ru4l) [inline packages/services/api/src/modules/audit-logs/module.graphql.ts:13]: We follow the convention of having a single input field named after the mutation field for Mutation fields.

File: packages/services/api/src/modules/audit-logs/module.graphql.ts [hunk 3]
```diff
@@ -1,56 +1,22 @@
...
-    node: AuditLog!
-    cursor: String!
-  }
-
-  type AuditLogPageInfo {
-    hasNextPage: Boolean!
-    hasPreviousPage: Boolean!
-    startCursor: String
-    endCursor: String
-  }
```
Comment (n1ru4l) [inline packages/services/api/src/modules/audit-logs/module.graphql.ts:52]: These are unused.

File: packages/services/api/src/modules/audit-logs/providers/audit-logs-manager.ts [hunk 4]
```diff
@@ -45,27 +44,25 @@ export class AuditLogManager {
...
 
     this.logger.info('Getting audit logs (organizationId=%s, filter=%o)', organizationId, filter);
-    const where: SqlValue[] = [];
-    where.push(sql`organization_id = ${organizationId}`);
-
-    const from = formatToClickhouseDateTime(startOfDay(filter.startDate));
-    const to = formatToClickhouseDateTime(endOfDay(filter.endDate));
-    where.push(sql`timestamp >= ${from} AND timestamp <= ${to}`);
-
-    const whereClause = where.length > 0 ? sql`WHERE ${sql.join(where, ' AND ')}` : sql``;
```
Comment (n1ru4l) [inline packages/services/api/src/modules/audit-logs/providers/audit-logs-manager.ts:55]: These can easily be inlined to simplify complexity and remove unnecessary condition checks that are always truthy (`where.length > 0`).

File: packages/services/api/src/modules/audit-logs/providers/audit-logs-manager.ts [hunk 5]
```diff
@@ -235,19 +234,7 @@ export class AuditLogManager {
...
-  async maskTokenForAuditLog(token: string, visibleStart = 3, visibleEnd = 3): Promise<string> {
-    if (token.length <= visibleStart + visibleEnd) {
-      return token;
-    }
-    return (
-      token.slice(0, visibleStart) +
-      '*'.repeat(token.length - visibleStart - visibleEnd) +
-      token.slice(-visibleEnd)
-    );
-  }
```
Comment (n1ru4l) [inline packages/services/api/src/modules/audit-logs/providers/audit-logs-manager.ts:252]: There is no need to re-declare this function here if it is already available everywhere by importing from `@hive/service-common`

File: packages/services/api/src/modules/audit-logs/resolvers/AuditLogIdRecord.ts
Comment (n1ru4l) [inline packages/services/api/src/modules/audit-logs/resolvers/AuditLogIdRecord.ts:1]: This file is unused

File: packages/services/api/src/modules/cdn/providers/cdn.provider.ts [hunk 6]
```diff
@@ -26,7 +26,6 @@ export class CdnProvider {
     logger: Logger,
     private session: Session,
     private auditLog: AuditLogRecorder,
-    private auditLogManager: AuditLogManager,
```
Comment (n1ru4l) [inline packages/services/api/src/modules/cdn/providers/cdn.provider.ts:29]: No need to inject the `AuditLogManager` for this function, as we can just use the `maskToken` function from `@hive/service-common`.

File: packages/services/api/src/modules/integrations/index.ts [hunk 7]
```diff
@@ -10,5 +9,5 @@ export const integrationsModule = createModule({
   dirname: __dirname,
   typeDefs,
   resolvers,
-  providers: [SlackIntegrationManager, GitHubIntegrationManager, AuditLogManager],
+  providers: [SlackIntegrationManager, GitHubIntegrationManager],
```
Comment (n1ru4l) [inline packages/services/api/src/modules/integrations/index.ts:12]: No need to to add `AuditLogManager`, since we removed the modules dependency upon it.

**Review by TuvalSimha — APPROVED** [review]

**Pull request merged**